### DISCUSSION TITLE
Output of the live video

### DISCUSSION THREAD

**Discussion by tataramazan** [vladmandic/human#346] [Q&A]
Can I get the output for the live video? For example, the output of the emotional detection of the live video.

**Comment by vladmandic** [comment]
i've moved this from issues to discussions as its a question and not a product isssue.

anyhow, short answer is yes.
longer answer would depend on your environment - is this browser or nodejs environment? in either case, there are examples how to process either video or webcam inputs (and yes, that is a most common use case - a processing of a live video).

**Reply by tataramazan** [reply]
I want to get the results of emotional detection status during the time in a live video. Can this be done over the web browser?

**Reply by vladmandic** [reply]
i'm still not sure what is the question as that is exactly what built-in demos showcase? obvious answer would be yes unless i'm misunderstanding something.

**Comment by tataramazan** [comment]
Sorry, I'm not a software developer. I didn't ask my question correctly. But with your software I want to do this: I want to measure the mood of the participants in 2 min. Overall, I want to measure the results of a person's emotional state for 2 minutes. So how can ı get the results in the web browser 
Thanks for help

**Reply by vladmandic** [reply]
again, this is pretty much out-of-the-box. you provide input video to `human` and look at the results - emotion for that point in time is included. then just store it anywhere you want and continue with detection.

**Comment by tataramazan** [comment]
OK. Thanks. ı have one more question. Maybe I missed it. What is the accuracy rate of emotional detection?

Thanks.

**Reply by vladmandic** [reply]
it heavily depends on quality of inputs and some additional tunable values (still exploring best defaults) such as how tightly to crop the detected face, but in general its over 80% although some may be mis-qualified (e.g. disgust is difficult to separate from anger).

**Comment by tataramazan** [comment]
Can we say the same for the video?

**Comment by vladmandic** [comment]
i said inputs because inputs can be be anything, including video. again, this is pretty much covered in docs and demos as basic use-case, really not sure what is the question.

**Comment by nicoeiris11** [comment]
@tataramazan In my case `face.scale=1.4`, `face.emotion.crop=0.1`, and `face.emotion.modelPath='affectnet-mobilenet.json'` was the most accurate setup.
